vignettes/synapser.Rmd

```diff
@@ -306,11 +365,11 @@ TODO
 synDelete(project)
 ```
 
-## File Views
+## <a name="File_Views"></a>File Views
```

Comment:
Yes, this is a large enough issue that it's probably best to make a separate vignette.  The vignette would cover views in general as well as file views.